app/code/core/Mage/Api/Model/Server/Handler/Abstract.php

```diff
@@ -35,9 +35,9 @@ public function __construct()
      * @param string $errorFile
      * @return bool
      */
-    public function handlePhpError($errorCode, $errorMessage, $errorFile)
+    public function handlePhpError($errorCode, $errorMessage, $errorFile, $errLine)
     {
-        Mage::log($errorMessage . $errorFile);
+        Mage::log($errorMessage . '  in ' . $errorFile . ' on line ' . $errLine);
```

Comment:
```suggestion
        Mage::log($errorMessage . ' in ' . $errorFile . ' on line ' . $errLine, Zend_Log::ERR);
```

Reply:
The double spaces is not an error.
PHP add also two spaces here, see also here:

https://github.com/OpenMage/magento-lts/blob/0ac3f19a23eed02fbb27cc5586254a5eb7d30af9/app/code/core/Mage/Core/functions.php#L182

But it's not really important, and phpdoc was not updated 😸.